### DISCUSSION TITLE
Do not suggest a note if that not is already linked in the file

### DISCUSSION THREAD

**Discussion by Richie-Peak** [brianpetro/obsidian-smart-connections#493] [Ideas]
If I have a note with a few links say [[example1]], [[example2]], [[example3]]. 

Is there a reason the plugin will often still recommend [[example1]], [[example2]], [[example3]] as potential smart connections? It seems intuitive to me that they'd be excluded.

**Comment by brianpetro** [comment]
@Richie-Peak thanks for pointing this out.

Currently, the behavior you're describing is how it's designed. But I think that your expectation is really intuitive.

So, I think the best way forward is to make an option to prevent already-linked notes from appearing in the Smart View results.

I initiated a feature request here https://github.com/brianpetro/obsidian-smart-connections/issues/494

Thanks for your help in improving Smart Connections!

🌴
